src/features/offer/components/SameArtistPlaylist/api/fetchOffersByArtist.test.ts

```diff
@@ -0,0 +1,41 @@
+import algoliasearch from 'algoliasearch'
+
+import { fetchOffersByArtist } from 'features/offer/components/SameArtistPlaylist/api/fetchOffersByArtist'
+import { offerAttributesToRetrieve } from 'libs/algolia/fetchAlgolia/buildAlgoliaParameters/offerAttributesToRetrieve'
+
+jest.mock('algoliasearch')
+
+const mockInitIndex = algoliasearch('', '').initIndex
+const search = mockInitIndex('').search as jest.Mock
+
+describe('fetchOffersByArtist', () => {
+  it('should execute the query if both artist and ean are provided', async () => {
```

Comment:
`should execute the query when artist and ean are provided`

Reply:
Je pense que @YassinL  l'avait bien écrit celui-ci @cedricls1 . C'est plus anglais :D

Reply:
Pas d'accord mais c'était une suggestion donc à lui de voir